front/src/app/core/save-indicator/save-indicator.component.ts

```diff
@@ -0,0 +1,49 @@
+import {Component, Input} from '@angular/core';
+import {CommonModule} from '@angular/common';
+import {SaveStatus} from '../types/save-status.types';
+
+@Component({
+  selector: 'app-save-indicator',
+  standalone: true,
+  imports: [CommonModule],
+  templateUrl: './save-indicator.component.html',
+  styleUrl: './save-indicator.component.scss'
+})
+export class SaveIndicatorComponent {
+  @Input() status: SaveStatus = 'idle';
+  @Input() savingMessage: string = 'Saving changes...';
+  @Input() savedMessage: string = 'Changes saved!';
+  @Input() errorMessage: string = 'Error saving changes';
+  @Input() position: 'bottom-right' | 'bottom-left' | 'top-right' | 'top-left' = 'bottom-right';
```

Comment:
suggestion: utilise des Signal Inputs (stable depuis Angular 19) https://www.dolmen.tools/en/angular/caniuse/explorer?search=inputs

```suggestion
  status = input<SaveStatus>('idle');
  savingMessage = input('Saving changes...');
  savedMessage = input('Changes saved!');
  errorMessage = input('Error saving changes');
  position = signal<'bottom-right' | 'bottom-left' | 'top-right' | 'top-left'>('bottom-right');
```